tests/phpunit/class-wc-stripe-checkout-sessions-ajax-handler-test.php

```
@@ -356,6 +356,124 @@ public function provide_test_update_checkout_session(): array {
+			remove_filter( 'wc_stripe_request_body', $capture_body, 10, 2 );
+			delete_option( 'woocommerce_checkout_phone_field' );
+			Ajax_Test_Helper::remove_hooks();
+		}
+
+		$this->assertIsArray( $captured_request );
+
+		if ( $expect_phone_collection ) {
+			$this->assertArrayHasKey( 'phone_number_collection', $captured_request );
+		} else {
+			$this->assertArrayNotHasKey( 'phone_number_collection', $captured_request );
+		}
```

PHPMD (2.15.0)

[error] The method test_phone_number_collection_based_on_wc_setting uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)